guide/C/ch_invest.xml

```diff
@@ -1605,7 +1605,7 @@ Income
             <row>
               <entry>Assets:Brokerage Account:Stock:AMZN</entry>
 
-              <entry>-100</entry>
+              <entry>&mdash;100</entry>
```

Comment:
I don't think use of - is a mistake, I just think &mdash; is clearer and therefore better for people who are learning.
As you don't agree, I have reverted to -.